r/src/array_to_vector.cpp

```diff
@@ -920,7 +920,7 @@ class Converter_Decimal : public Converter {
 
   Status Ingest_some_nulls(SEXP data, const std::shared_ptr<arrow::Array>& array,
                            R_xlen_t start, R_xlen_t n, size_t chunk_index) const {
-    using DecimalArray = typename TypeTraits<Type>::ArrayType;
+    //using DecimalArray = typename TypeTraits<Type>::ArrayType;
```

Comment:
It should actually be used in `const auto& decimals_arr = checked_cast<const DecimalArray&>(*array);`

Reply:
Done!